tally_ho/apps/tally/management/commands/import_data.py

```diff
@@ -318,19 +318,73 @@ def process_results_form_row(tally, row, command=None, logger=None):
     ballot_number, code, station_number, gender, name,\
         office_name, _, barcode, serial_number = row[0:9]
 
-    ballot = Ballot.objects.get(number=ballot_number, tally=tally)
     gender = gender and getattr(Gender, gender.upper())
+    ballot = None
+
+    try:
+        ballot = Ballot.objects.get(number=ballot_number, tally=tally)
+        if ballot and not ballot.active:
```

Comment:
I'm not sure it's possible to be in a scenario where `ballot` is null, wouldn't that have lead to a `DoesNotExist` exception?

Reply:
Nice catch, will change this